apps/terragrunt.hcl

```diff
@@ -77,7 +72,14 @@ generate "provider" {
   contents  = <<EOF
 ${local.providers.locals.azure_provider}
 
-provider "github" {}
+provider "github" {
+  owner = var.serve.github_owner
+  app_auth {
+    id              = var.serve.github_app_id
+    installation_id = var.serve.github_app_installation_id
+    pem_file        = file("./github.pem")
```

Comment:
if there's not a nice way to get this from an env var then I vote just living with the duplication in the gh workflow

Reply:
Turned out to be fairly easy so have added :)